src/Base/MultiFab.cpp

```diff
@@ -193,22 +337,69 @@ void init_MultiFab(py::module &m)
         )
 
         /* Constructors */
-        .def(py::init< >())
-        //.def(py::init< MultiFab && >())
-        .def(py::init< const BoxArray&, const DistributionMapping&, int, int >())
-        .def(py::init< const BoxArray&, const DistributionMapping&, int, int,
-                       MFInfo const & >())
+        .def(py::init< >(),
+            R"(Constructs an empty MultiFab.
+
+            Data can be defined at a later time using the define member functions
+            inherited from FabArray.)"
+        )
+        .def(py::init< Arena* >(),
+            py::arg("a"),
+            R"(Constructs an empty MultiFab.
+
+            Data can be defined at a later time using the define member functions.
+            If ``define`` is called later with a nullptr as MFInfo's arena, the
+            default Arena ``a`` will be used.  If the arena in MFInfo is not a
+            nullptr, the MFInfo's arena will be used.)"
+        )
+    ;
+
+    constexpr auto doc_mf_init = R"(Constructs a MultiFab.
+
+    The size of the FArrayBox is given by the Box grown by \p ngrow, and
+    the number of components is given by \p ncomp. If \p info is set to
+    not allocating memory, then no FArrayBoxes are allocated at
+    this time but can be defined later.
+
+    Parameters
+    ----------
+    bxs :
+      a valid region
+    dm :
+      a DistribuionMapping
+    ncomp :
+      number of components
+    ngrow :
+      number of cells the region grows
+    info :
+      MultiFab info, including allocation Arena
+    factory :
+      FArrayBoxFactory for embedded boundaries)";
+
+    py_MultiFab
         .def(py::init< const BoxArray&, const DistributionMapping&, int, int,
-                       MFInfo const &, FabFactory<FArrayBox> const & >())
+                       MFInfo const &, FabFactory<FArrayBox> const & >(),
+            py::arg("bxs"), py::arg("dm"), py::arg("ncomp"), py::arg("ngrow"),
+#ifdef AMREX_STRICT_MODE
+            py::arg("info"), py::arg("factory"),
+#else
+            py::arg("info") = MFInfo(),
+            py::arg("factory") = FArrayBoxFactory(),
```

Comment:
```
ImportError: arg(): could not convert default argument 'factory: amrex::DefaultFabFactory<amrex::FArrayBox>' in method '<class 'amrex.space1d.amrex_1d_pybind.MultiFab'>.__init__' into a Python object (type not registered yet?)
```

Reply:
I think in Python, such default constructor _calls_ do generally not work (and would only create an object once at import time). Will remove those here.